app/js/ffxivcraftmodel.js

```diff
@@ -73,67 +73,58 @@ Synth.prototype.calculateBaseProgressIncrease = function (levelDifference, craft
     var baseProgress = 0;
     var levelCorrectionFactor = 0;
     var levelCorrectedProgress = 0;
+    var recipeLevelPenalty = 0;
 
-    if (crafterLevel >= 260) {
-        baseProgress = 1.834712812e-5 * craftsmanship * craftsmanship + 1.904074773e-1 * craftsmanship + 1.544103837;
-    }
-    else if (crafterLevel >= 120) {
-        baseProgress = 2.09860e-5 * craftsmanship * craftsmanship + 0.196184 * craftsmanship + 2.68452;
-    }
-    else if (crafterLevel < 120) {
-        baseProgress = 0.214959 * craftsmanship + 1.6;
-    }
+    baseProgress = 1.834712812e-5 * craftsmanship * craftsmanship + 1.904074773e-1 * craftsmanship + 1.544103837;
```

Comment:
Have you been able to test this with lower level crafters?

Reply:
Yea, there was a bunch of 1-50 data in various issues on here I was able to test against. It was within a point for all the ones I could find.